yaksh/templates/yaksh/login.html

```diff
@@ -24,29 +81,58 @@ <h3> Login with </h3>
         </a>
         </form>
         </div>
-        <div class="col-md-6">
-            <p> Online Test Interface for conducting online programming quiz.
-            <ul>
-                <li>Define fairly complicated programming problems and have users
-                solve the problem. </li>
-                <li>Immediate verification of code solution. </li>
-                <li>Supports pretty much arbitrary programming questions in Python, C, C++ and
-                simple Bash and uses "test cases" to test the implementations of the students.</li>
-                <li>Supports simple multiple choice questions and File uploads.</li>
-                <li>Since it runs on your Python, you could technically test any Python based library.</li>
-                <li>Scales to over 500+ simultaneous users.</li>
-            </ul>
-            </p>
-            <br/>
-            <p><b>Fork us at:</b>
-            <a class = "btn btn-social-icon btn-github"
-            href ="https://github.com/fossee/online_test"
-            style="color:black">
-            
-            <span class="fa fa-github" style="font-size:48px"></span>
-            </p>
-            </a>
+        </br>
+        <div class="yakshred h4">
+            Best platform you can find to learn</br> and to test your skills
+        </div>
         </div>
+        
+    </div>
+
+    <div>
+        </br></br>
+        <h2 class="h4 yakshred" id="aboutus">About Us</h2>
+        <div class="yakshwell">
+        <h5><b>History</b></h5>
+
+        <p>At FOSSEE, Nishanth had implemented a nice django based app to test for multiple-choice questions. Prabhu Ramachandran was inspired by a programming contest that he saw at PyCon APAC 2011. Chris Boesch, who administered the contest, used a nice web application Singpath that he had built on top of GAE that basically checked your Python code, live. This made it fun and interesting.</p>
+        <p>Prabhu wanted an implementation that was not tied to GAE and hence wrote the initial cut of what is now 'Yaksh'. The idea being that anyone can use this to test students programming skills and not have to worry about grading their answers manually and instead do so on their machines.</p>
+        <p>The application has since been refactored and maintained by FOSSEE Developers.</p>
+
+        <h5><b>Contact</b></h5>
+        <p>For further information and support you can contact 
+        Python Team at FOSSEE: pythonsupport@fossee.in</p>
+
+        <h5><b>License</b></h5>
+        <p>This is distributed under the terms of the BSD license. Copyright information is at the bottom of this file.</p>
+
+        <h5><b>Authors</b></h5>
+        <p><a href="https://github.com/FOSSEE/online_test/graphs/contributors">FOSSEE Developers</a><br>
+        Copyright (c) 2011-2017 FOSSEE</p>
```

Comment:
This should be updated to 2018. Also, the current year could just be displayed from the Django `time` templatetag.

Reply:
Done